server/util/db/db.go

```diff
@@ -193,7 +194,18 @@ func openDB(configurator *config.Configurator, dialect string, connString string
 	}
 
 	var l logger.Interface
-	l = sqlLogger{Interface: logger.Default, logLevel: logger.Warn}
+	// This is the same as logger.Default, but with colors turned off (since the
+	// output may not support colors) and sending output to stderr (to be
+	// consistent with the rest of our logs).
+	// TODO: Have all logs written to zerolog instead.
+	gormLogger := logger.New(
+		golog.New(os.Stderr, "\r\n", golog.LstdFlags),
+		logger.Config{
+			SlowThreshold: 200 * time.Millisecond,
+			LogLevel:      logger.Warn,
+			Colorful:      false,
```

Comment:
Can we leave the colors on for local runs? 
We could make this conditional on the "structured logging" config option.

Reply:
Done